### PR TITLE
fix: block view and port display

### PR DESCRIPTION
**Pull request by ycjcl868**
Close https://github.com/umijs/umi/issues/4346
Close https://github.com/ant-design/ant-design-pro/issues/6275

### CODE REVIEW

File: packages/plugin-ui-blocks/src/ui/server/util/index.ts [hunk 1]
```diff
@@ -143,7 +143,7 @@ export const DEFAULT_RESOURCES: Resource[] = [
       fetchCDNBlocks({
         pkg: '@umijs/assets-umi',
         summary: 'templates.json',
-        version: '^1.0.0',
+        version: '~1.0.0',
```
Comment (ycjcl868) [inline packages/plugin-ui-blocks/src/ui/server/util/index.ts:146]: 先定个版本，后面数据结构有变化

File: packages/plugin-ui-blocks/src/sdk/flagBabelPlugin/index.ts [hunk 2]
```diff
@@ -164,7 +164,7 @@ export default () => {
    * @param filename 路径名
    */
   const checkPathFilename = (filename: string): boolean => {
-    if (filename.indexOf('/pages/') > -1 || filename.indexOf('/page/') > -1) {
+    if (winPath(filename).indexOf('/pages/') > -1 || winPath(filename).indexOf('/page/') > -1) {
```
Comment (ycjcl868) [inline packages/plugin-ui-blocks/src/sdk/flagBabelPlugin/index.ts:167]: https://github.com/ant-design/ant-design-pro/issues/6275

File: packages/plugin-ui-blocks/ui/BlocksViewer.tsx [hunk 3]
```diff
@@ -314,8 +253,65 @@ const BlocksViewer: React.FC<Props> = props => {
         </Tooltip>,
       ]);
     }
+  }, [current]);
```
Comment (ycjcl868) [inline packages/plugin-ui-blocks/ui/BlocksViewer.tsx:256]: 右上角按钮国际化不显示的问题

File: packages/plugin-ui-blocks/src/sdk/flagBabelPlugin/index.ts (hunk 2 above)
Comment (chenshuai2144) [inline packages/plugin-ui-blocks/src/sdk/flagBabelPlugin/index.ts:167]:
```suggestion
    if (winPath(filename).indexOf('pages/') > -1 || winPath(filename).indexOf('page/') > -1) {
```

**Pull request merged**